docs/integrations/terraform/introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Statsig Terraform Provider
+---
+
+The Statsig Terraform Provider allows you to configure your gates and experiments with Terraform. This is achieved by leveraging the Console API. If there is something you need to perform that isn't supported by the Terraform Provider, checkout the [Console API](https://docs.statsig.com/console-api/introduction).
```

Comment:
This is achieved by leveraging the Console API => The provider synchronizes with Statsig via the console API
Or something.  "This is achieved by leveraging" is just awkward